### ISSUE TITLE
Autoplay on angular 7

### ISSUE THREAD

**Issue by aparraga** [vitalii-andriiovskyi/ngx-owl-carousel-o#23]
Hi,

The carousel is always in autoplay mode in angular 7, even if I set autoplay: false.
My configuration is:
```
        this.customOptions = {
            loop: false,
            checkVisible: false,
            rewind: false,
            margin: 10,
            nav: false,
            dots: false,
            slideBy: this.visibleNumberOfProducts,
            responsive: {
                0: {
                    items: 2
                },
                600: {
                    items: 3
                },
                1000: {
                    items: this.visibleNumberOfProducts
                }
            }
        };
```

Not sure what I'm doing wrong...

**Comment by hughjdavey** [comment]
I am also having this problem... options seem to be ignored. I have nav=false and dots=true, yet a Prev/Next nav appears

**Comment by vitalii-andriiovskyi** [comment]
Hi, 
Sorry for the late answer.
If your issue is still actual, please, provide more details about the usage of the carousel in your project. It'll help me to reproduce your case and see what's wrong.

**Comment by craftball** [comment]
It seems like everytime I hover the mouse away from the carousel into another div element the slides will move by 1 even if autoplay is false

**Comment by vitalii-andriiovskyi** [comment]
Hi, @craftball.

Thanks for pointing out the detail of the problem. I solved it and published the new version of the lib: `v1.1.1`.
